### DISCUSSION TITLE
Welcome to Public Discussions!

### DISCUSSION THREAD

**Discussion by BS-SNS** [BS-SNS/Public#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gdreinert** [comment]
We are happy to announce the first talk in our seminar series: 

Day: June 12, 2023, 12-13 UTC (13-14 London Time)  

Speaker: Eric Kolaczyk (McGill) 

Title: Coevolving latent space network with attractors models for polarization

Abstract: A broadly applicable class of coevolving latent space network with attractors (CLSNA) models is developed, where nodes represent individual social actors assumed to lie in an unknown latent space, edges represent the presence of a specified interaction between actors, and attractors are added in the latent level to capture the notion of attractive and repulsive forces. We apply the CLSNA models to understand the dynamics of partisan polarization on social media, where we expect US Republicans and Democrats to increasingly interact with their own party and disengage with the opposing party. Using longitudinal social networks from the social media platforms Twitter and Reddit, we investigate the relative contributions of positive (attractive) and negative(repulsive) forces among political elites and the public, respectively. Our goals are to disentangle the positive and negative forces within and between parties and explore if and how they change over time. Our analysis confirms the existence of partisan polarization in social media interactions among both political elites and the public. Moreover, while positive partisanship is the driving force of interactions across the full periods of study for both the public and Democratic elites, negative partisanship has come to dominate Republican elites interactions since the run-up to the 2016 presidential election. 

This is joint work with Xiaojing Zhu, Cantay Caliskan, Dino Christenson, Dylan Walker, and Kostas Spiliopoulos.
